task-list-rails/app/models/person.rb

```diff
@@ -0,0 +1,17 @@
+class Person < ActiveRecord::Base
+  has_many :tasks
+
+  def self.num_completed(person_id)
+    person = Person.find(person_id)
+    tasks = person.tasks
+    completed = tasks.where(completed: true)
+    return completed.length
+  end
+
+  def self.num_not_completed(person_id)
```

Comment:
One cool thing about Ruby is that you can write a method that returns two values. I think since these two methods are so similar to one another and require some of the same code and are used in the same place in the view... it could be worthwhile to try to combine those and return the two values in an array.